chains/zetachain/addresses.yaml

```diff
@@ -5,7 +5,7 @@ fallbackRoutingHook: "0x8F1E22d309baa69D398a03cc88E9b46037e988AA"
 interchainAccountIsm: "0x2b6d3F7d28B5EC8C3C028fBCAdcf774D9709Dd29"
 interchainAccountRouter: "0x3AdCBc94ab8C48EC52D06dc65Bb787fD1981E3d5"
 interchainGasPaymaster: "0x931dFCc8c1141D6F532FD023bd87DAe0080c835d"
-interchainSecurityModule: "0xCAc4123018406911D43e778410313A594276c96E"
+interchainSecurityModule: "0x858a077945bCC4afA40DaE5a75faB9237e899b30"
```

Comment:
this address shows up as an EOA, I assume it's an explorer failure? https://explorer.zetachain.com/address/0x858a077945bCC4afA40DaE5a75faB9237e899b30

Reply:
yup! Whenever in doubt, use the CLI's ISM read command:

```
$ hyperlane ism read --chain zetachain --address 0x858a077945bCC4afA40DaE5a75faB9237e899b30

Hyperlane CLI
{"level":30,"time":1726252916458,"pid":65626,"msg":"Your CLI version: 5.1.0, latest version: 5.1.2"}
Hyperlane ISM Read
------------------
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
ISM Config at 0x858a077945bCC4afA40DaE5a75faB9237e899b30 on zetachain:
address: "0x858a077945bCC4afA40DaE5a75faB9237e899b30"
type: staticAggregationIsm
modules:
  - address: "0x7b75b29caD47e10146e29BBf7BD9025e021a7023"
    owner: "0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba"
    type: pausableIsm
    paused: false
  - owner: "0x9d399876522Fc5C044D048594de399A2349d6026"
    address: "0x92Eb9aC5C7658c4c6ea45dfA1C91122E844a2e88"
    type: domainRoutingIsm
    domains:
      bitlayer:
        address: "0xCC44290565F1d90b51adb574157Bb9A0571bCE39"
        type: staticAggregationIsm
        modules:
          - address: "0x3f85131B9C28beA465FB4A9570D4437b01E6eA3A"
            type: merkleRootMultisigIsm
            validators:
              - "0x1d9B0F4EA80dbFC71cB7d64d8005Eccf7C41e75f"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x8f924deCc739778eDac74b144e30a42aece4Ad95"
            type: messageIdMultisigIsm
            validators:
              - "0x1d9B0F4EA80dbFC71cB7d64d8005Eccf7C41e75f"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      ancient8:
        address: "0xBcDd450cB8105465A91C0b80f164A4F8DCdc9D99"
        type: staticAggregationIsm
        modules:
          - address: "0x0427e0084e8BEC8c3D7Cc2a31DB9ACA8Fc0d3301"
            type: merkleRootMultisigIsm
            validators:
              - "0x95C7bF235837cb5A609fe6c95870410B9f68bCFF"
              - "0xa5a56e97fB46f0AC3A3D261E404ACB998d9a6969"
              - "0xBB5842aE0e05215b53Df4787A29144EfB7e67551"
            threshold: 2
          - address: "0x10F5d065b6b212FEC85DDB36A8e8239c40F3C209"
            type: messageIdMultisigIsm
            validators:
              - "0x95C7bF235837cb5A609fe6c95870410B9f68bCFF"
              - "0xa5a56e97fB46f0AC3A3D261E404ACB998d9a6969"
              - "0xBB5842aE0e05215b53Df4787A29144EfB7e67551"
            threshold: 2
        threshold: 1
      cyber:
        address: "0x20870AEC44944149C6348A1C278e6e62B6970282"
        type: staticAggregationIsm
        modules:
          - address: "0x2e4bA3D3ADCCE1691ae04745e1932BF49f8D9e0c"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x94D7119ceeB802173B6924E6cC8c4CD731089a27"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x5b5C1587ccD65a4594D9850e0c1B95eA193aCC06"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x94D7119ceeB802173B6924E6cC8c4CD731089a27"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      astarzkevm:
        address: "0xf07706c1851b8D10DC6F20309E7E619691BE529e"
        type: staticAggregationIsm
        modules:
          - address: "0x2eF19BAA871145239B6B2EF53C601fCf40D2040C"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x89EcdD6caf138934bF3A2Fb7B323984d72fd66dE"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x3ED3489E58a8249847256090cCc06e9002E6796A"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x89EcdD6caf138934bF3A2Fb7B323984d72fd66dE"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      dogechain:
        address: "0x47bD9591E171C991E45e82a123bbA4C0546E07B6"
        type: staticAggregationIsm
        modules:
          - address: "0x811cd48c1C7c4Cb2DA73fC70e694fe320151aBf5"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xE43F742C37858746E6d7E458Bc591180D0CbA440"
            threshold: 2
          - address: "0x92EbCc163063FeD280c428d8f940682C33c85dC4"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xE43F742C37858746E6d7E458Bc591180D0CbA440"
            threshold: 2
        threshold: 1
      arbitrum:
        address: "0x7EC7b8F554ef91a3669705557d68d1123E50d5D8"
        type: staticAggregationIsm
        modules:
          - address: "0x0E4CBdcf36A4e3c0653977C5Fc9816ad0e0FEb2C"
            type: merkleRootMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x4d966438fe9E2B1e7124c87bBB90cB4F0F6C59a1"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xec68258A7c882AC2Fc46b81Ce80380054fFB4eF2"
            threshold: 3
          - address: "0xcCd3580D484d6b444E9D39AE5bdB2362D5C72A7A"
            type: messageIdMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x4d966438fe9E2B1e7124c87bBB90cB4F0F6C59a1"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xec68258A7c882AC2Fc46b81Ce80380054fFB4eF2"
            threshold: 3
        threshold: 1
      ethereum:
        address: "0xbC91C144F694923B55aD40054A4E47B8041356A1"
        type: staticAggregationIsm
        modules:
          - address: "0x2c5C5517f0D84d0667A5F32152363B1841f37a77"
            type: messageIdMultisigIsm
            validators:
              - "0x03c842db86A6A3E524D4a6615390c1Ea8E2b9541"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x94438A7De38D4548ae54dF5c6010c4Ebc5239eAE"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xb683B742B378632A5F73a2a5a45801b3489Bba44"
              - "0xBf1023EFF3dBA21263bf2db2adD67A0d6bcdA2dE"
            threshold: 4
          - address: "0xAF5DBBebb74e59A5C47b05dD9172Dc94000aaE8C"
            type: merkleRootMultisigIsm
            validators:
              - "0x03c842db86A6A3E524D4a6615390c1Ea8E2b9541"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x94438A7De38D4548ae54dF5c6010c4Ebc5239eAE"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xb683B742B378632A5F73a2a5a45801b3489Bba44"
              - "0xBf1023EFF3dBA21263bf2db2adD67A0d6bcdA2dE"
            threshold: 4
        threshold: 1
      blast:
        address: "0x3172AC7d022ffCE7441bB3561226a27A7441817A"
        type: staticAggregationIsm
        modules:
          - address: "0x02c03689091291b580Dd599826dA94f82B79B410"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xaE53467A5c2A9D9420c188D10FEF5e1d9B9A5B80"
              - "0xF20c0b09F597597c8d2430d3d72DFDdaf09177d1"
            threshold: 2
          - address: "0xE992DD88Af8534F22ebE2d77fa2eFC672378DE66"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xaE53467A5c2A9D9420c188D10FEF5e1d9B9A5B80"
              - "0xF20c0b09F597597c8d2430d3d72DFDdaf09177d1"
            threshold: 2
        threshold: 1
      avalanche:
        address: "0xba323ea4Af99f0c4DdFD66994A4fAD813b2253D2"
        type: staticAggregationIsm
        modules:
          - address: "0x2bEde7C1F11547E313d63E4d56e4bA74E98D2957"
            type: messageIdMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x3fB8263859843bfFb02950c492D492cAe169f4Cf"
              - "0x402E0f8c6E4210D408B6AC00D197D4A099fcd25A"
            threshold: 2
          - address: "0x4D310777D35B1607E97cdb503C4Dcdd0bF1078D3"
            type: merkleRootMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x3fB8263859843bfFb02950c492D492cAe169f4Cf"
              - "0x402E0f8c6E4210D408B6AC00D197D4A099fcd25A"
            threshold: 2
        threshold: 1
      coredao:
        address: "0x1d5921c73C1299a1b349B84C440666e641e4Ee7A"
        type: staticAggregationIsm
        modules:
          - address: "0x1b490DeD49a3cfFee9647e2f9f11Ce39A6ED46B4"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xbd6E158a3F5830D99d7d2bCE192695Bc4a148dE2"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x3D74a439E20Bc858187080D79449d48D93898b41"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xbd6E158a3F5830D99d7d2bCE192695Bc4a148dE2"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      endurance:
        address: "0xcf68bfc3845F52bb4df6FbA97CA7E17657939D12"
        type: staticAggregationIsm
        modules:
          - address: "0xcf529AA5422Eb8c150e7EfeA498e4eaBD1B5b270"
            type: merkleRootMultisigIsm
            validators:
              - "0x28c5B322da06F184Ebf68693c5d19df4d4aF13E5"
              - "0x7419021C0de2772B763E554480158A82A291C1F2"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xebeF2458477fb70d1a204740AeF82358dC6015f1"
            type: messageIdMultisigIsm
            validators:
              - "0x28c5B322da06F184Ebf68693c5d19df4d4aF13E5"
              - "0x7419021C0de2772B763E554480158A82A291C1F2"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      celo:
        address: "0xf9C36B05601D111eCC0f4d981dE48C280EbBbeDE"
        type: staticAggregationIsm
        modules:
          - address: "0x69b0c456DDc0D1605CAD6922349EdE8E071D965d"
            type: messageIdMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x622e43BAF06ad808cA8399360D9a2d9A1a12688b"
              - "0x63478422679303c3e4fC611B771fa4a707eF7f4a"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
            threshold: 3
          - address: "0xaE1Aef3F11B8F097e54FFd7f1A892f1DD948bc9F"
            type: merkleRootMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x622e43BAF06ad808cA8399360D9a2d9A1a12688b"
              - "0x63478422679303c3e4fC611B771fa4a707eF7f4a"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
            threshold: 3
        threshold: 1
      cheesechain:
        address: "0xB7cb1654568dCC64f52AF72391F8948564AC4C02"
        type: staticAggregationIsm
        modules:
          - address: "0xAef0E4A32A22861EFC7aFFbc1F641e60423096D7"
            type: merkleRootMultisigIsm
            validators:
              - "0x478FB53c6860AE8fC35235bA0d38d49B13128226"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xF78DA66dcb355dDbF0cC23573F42A9Ffd183badd"
            type: messageIdMultisigIsm
            validators:
              - "0x478FB53c6860AE8fC35235bA0d38d49B13128226"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      bsc:
        address: "0x5AF06b8f34F8E954056A2f6591da2268a63dc358"
        type: staticAggregationIsm
        modules:
          - address: "0x7a2dBb0360A3D2467381923fEEA74618CF8407AD"
            type: merkleRootMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x570AF9B7B36568C8877eeBBA6c6727aA9DAB7268"
              - "0x8292B1A53907eCE0f76Af8A50724e9492bcDc8A3"
            threshold: 3
          - address: "0xEaA1bc260081e725CE3d95C82422B00600a147FA"
            type: messageIdMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x570AF9B7B36568C8877eeBBA6c6727aA9DAB7268"
              - "0x8292B1A53907eCE0f76Af8A50724e9492bcDc8A3"
            threshold: 3
        threshold: 1
      base:
        address: "0x28911A97B04D2660a72F7E4ace2CaF1B0745106D"
        type: staticAggregationIsm
        modules:
          - address: "0x951276eEd531219Dc5Cc495E3aEdddFCcbD42E2d"
            type: messageIdMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xB9453D675E0Fa3c178A17B4cE1AD5B1A279B3aF9"
              - "0xCff391b4E516452d424db66beb9052b041a9eD79"
            threshold: 3
          - address: "0xc0b6d83F25f5d766410e4599f27cca9F36e5380A"
            type: merkleRootMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xB9453D675E0Fa3c178A17B4cE1AD5B1A279B3aF9"
              - "0xCff391b4E516452d424db66beb9052b041a9eD79"
            threshold: 3
        threshold: 1
      degenchain:
        address: "0x9984446b99BB1D386e776CB68BEf60624EE59307"
        type: staticAggregationIsm
        modules:
          - address: "0x12300447C0B31ef3c526fcf6433c3647AdcA2D43"
            type: messageIdMultisigIsm
            validators:
              - "0x433E311f19524Cd64FB2123Ad0aa1579a4e1Fc83"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x49D057c63057FEfE7343A4Bda02B1f026bFfdAd1"
            type: merkleRootMultisigIsm
            validators:
              - "0x433E311f19524Cd64FB2123Ad0aa1579a4e1Fc83"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      everclear:
        address: "0x6622341C77f1D72906483EE35A7EC39fbc001276"
        type: staticAggregationIsm
        modules:
          - address: "0xf38a22798C3a9b694Afab960306313FC264b93c4"
            type: merkleRootMultisigIsm
            validators:
              - "0xD79DFbF56ee2268f061cc613027a44A880f61Ba2"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xeFf20AE3d5AB90aBb11E882CfCe4b92ea6c74837"
            threshold: 2
          - address: "0xf5d3Af6A3045b1EB0B4180C307FAFe82C34b915d"
            type: messageIdMultisigIsm
            validators:
              - "0xD79DFbF56ee2268f061cc613027a44A880f61Ba2"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xeFf20AE3d5AB90aBb11E882CfCe4b92ea6c74837"
            threshold: 2
        threshold: 1
      astar:
        address: "0x7D6C761Ac69bb6dcfc032F59E467548b32d85248"
        type: staticAggregationIsm
        modules:
          - address: "0x0A8c9756091Ec6e280263687c796Cb75B385F794"
            type: merkleRootMultisigIsm
            validators:
              - "0x4d1B2cAde01eE3493F44304653D8E352C66eC3E7"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x65D3E1D73EED6e63c05D133b1C2c3619E62Dd660"
            type: messageIdMultisigIsm
            validators:
              - "0x4d1B2cAde01eE3493F44304653D8E352C66eC3E7"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      bob:
        address: "0x1D5A031641689b75A5b9de7A76621135F49B0b70"
        type: staticAggregationIsm
        modules:
          - address: "0x2E74C95a0F304Db3A8322EcE5259dBb12b8d9A12"
            type: merkleRootMultisigIsm
            validators:
              - "0x20f283be1Eb0E81e22F51705dCB79883CFdD34aa"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x4B4a30c50d03CD09eF7326b7a34a12Cc68c70318"
            type: messageIdMultisigIsm
            validators:
              - "0x20f283be1Eb0E81e22F51705dCB79883CFdD34aa"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      eclipsemainnet:
        address: "0xD57E854b094D00fC9d3604F8f58386B789335927"
        type: staticAggregationIsm
        modules:
          - address: "0x07c79FC7a7e12cF305f315b5811547C5717E99D0"
            type: merkleRootMultisigIsm
            validators:
              - "0x3571223e745dC0fCbDEFa164C9B826B90c0d2DAc"
              - "0x4d4629F5bfeABe66Edc7A78da26Ef5273C266f97"
              - "0xEa83086a62617A7228ce4206FAe2ea8b0ab23513"
              - "0xebB52D7eaa3ff7A5A6260bfe5111CE52D57401d0"
            threshold: 3
          - address: "0x7E38c7aaFf51F22B0cf99661D7613e1C53fC6B91"
            type: messageIdMultisigIsm
            validators:
              - "0x3571223e745dC0fCbDEFa164C9B826B90c0d2DAc"
              - "0x4d4629F5bfeABe66Edc7A78da26Ef5273C266f97"
              - "0xEa83086a62617A7228ce4206FAe2ea8b0ab23513"
              - "0xebB52D7eaa3ff7A5A6260bfe5111CE52D57401d0"
            threshold: 3
        threshold: 1
      flare:
        address: "0xD108f20dcbAD729c8cD3a7c3e98C92c5d1Eb6a7c"
        type: staticAggregationIsm
        modules:
          - address: "0x59474F065da910c0f7Ff4288459912A2B20cD5A9"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xB65e52Be342DbA3Ab2C088CeEb4290C744809134"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xfA0685AD1eB92D9bED57EdC822Cc306e191b6c1C"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xB65e52Be342DbA3Ab2C088CeEb4290C744809134"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      kroma:
        address: "0xC9302CE3E0453992c1D770d2024FcF9F6aFeDc8D"
        type: staticAggregationIsm
        modules:
          - address: "0x45C77B82F32FdC54D99EED2aE762Db27cdDaBe67"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x71B83C21342787D758199e4b8634D3A15F02DC6E"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x57a15ea1866e44f32e9E64309D5d9e4b938Cef12"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x71B83C21342787D758199e4b8634D3A15F02DC6E"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      mode:
        address: "0x0172867a8c69F5F27440e102512Cb0227e64a660"
        type: staticAggregationIsm
        modules:
          - address: "0x7E1cfa781DfFf0007D096A503a478689b735A615"
            type: merkleRootMultisigIsm
            validators:
              - "0x101cE77261245140A0871f9407d6233C8230Ec47"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x7e29608C6E5792bBf9128599ca309Be0728af7B4"
              - "0x7EB2e1920A4166C19D6884c1CEc3d2CF356fc9B7"
            threshold: 3
          - address: "0xac870fb3CF63e272e7cd8819EC452cfDB8C561aA"
            type: messageIdMultisigIsm
            validators:
              - "0x101cE77261245140A0871f9407d6233C8230Ec47"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x7e29608C6E5792bBf9128599ca309Be0728af7B4"
              - "0x7EB2e1920A4166C19D6884c1CEc3d2CF356fc9B7"
            threshold: 3
        threshold: 1
      mantle:
        address: "0x86F4a0C3cf5A431a08D5eD87E73D4B605F1e6F66"
        type: staticAggregationIsm
        modules:
          - address: "0x293b055A9034EAD3fFf6BbBfdC4eF3C2f49aa992"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xf930636C5A1A8bf9302405F72e3aF3c96Ebe4a52"
            threshold: 2
          - address: "0xF6ea2c3Ec6b8989C8996Feec5E94d76B22EC30f5"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xf930636C5A1A8bf9302405F72e3aF3c96Ebe4a52"
            threshold: 2
        threshold: 1
      neutron:
        address: "0x32E905cF3480e7674087bF69b475e3E54166a1d3"
        type: staticAggregationIsm
        modules:
          - address: "0x4d7C0FAaB90FdAC50335e1De37BE8Fd3763CAc01"
            type: merkleRootMultisigIsm
            validators:
              - "0x42B6De2edBaA62C2eA2309Ad85D20B3E37d38Acf"
              - "0x42FA752DEFe92459370A052B6387a87f7DE9b80C"
              - "0x47AA126E05933b95c5eB90b26e6b668D84F4b25A"
              - "0x54B2cca5091B098A1A993dEC03c4d1EE9AF65999"
              - "0xA9b8c1f4998f781f958C63cfCD1708d02f004fF0"
              - "0xb65438a014FB05fbADCFE35bC6e25D372b6Ba460"
              - "0xC79503a3e3011535a9c60f6d21F76F59823A38bD"
            threshold: 4
          - address: "0x718E0bc33c50DB90ef27A97B37E4580c9Fa1bbF3"
            type: messageIdMultisigIsm
            validators:
              - "0x42B6De2edBaA62C2eA2309Ad85D20B3E37d38Acf"
              - "0x42FA752DEFe92459370A052B6387a87f7DE9b80C"
              - "0x47AA126E05933b95c5eB90b26e6b668D84F4b25A"
              - "0x54B2cca5091B098A1A993dEC03c4d1EE9AF65999"
              - "0xA9b8c1f4998f781f958C63cfCD1708d02f004fF0"
              - "0xb65438a014FB05fbADCFE35bC6e25D372b6Ba460"
              - "0xC79503a3e3011535a9c60f6d21F76F59823A38bD"
            threshold: 4
        threshold: 1
      molten:
        address: "0xaAAcF4c937eA91a719DF9950D64968f1d9690723"
        type: staticAggregationIsm
        modules:
          - address: "0x1811C7fAf2ca710F600eB2FBDf9365cb4ecBE926"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xAD5aa33F0D67f6FA258abBe75458eA4908f1dC9f"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x983B8dd95def2d0Fd1e8349CCE39347CFfad1C03"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xAD5aa33F0D67f6FA258abBe75458eA4908f1dC9f"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      fusemainnet:
        address: "0xa76fB14173587B065E3F79A5e0bb1915831c5911"
        type: staticAggregationIsm
        modules:
          - address: "0x24e47e2d58c28cD06F046823d8ca71864CA55277"
            type: messageIdMultisigIsm
            validators:
              - "0x6760226b34213d262D41D5291Ed57E81a68b4E0b"
              - "0x770c8ec9aaC8cEc4B2eaD583b49aCFBC5A1cF8a9"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x8B029D3254F3d859883eFB9DfbAEf01D05EF9638"
            type: merkleRootMultisigIsm
            validators:
              - "0x6760226b34213d262D41D5291Ed57E81a68b4E0b"
              - "0x770c8ec9aaC8cEc4B2eaD583b49aCFBC5A1cF8a9"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      metis:
        address: "0x75C15eb10C1dF1164a297fe7dcfa55f98a74dCdc"
        type: staticAggregationIsm
        modules:
          - address: "0x50E9ab0b4Fd19337D3C8069Eb37E703678216622"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xc4A3D25107060E800A43842964546DB508092260"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xf62818A375857f6D6fF8766De86Fd01e690C3730"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xc4A3D25107060E800A43842964546DB508092260"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      fraxtal:
        address: "0x9507f56C056AcE75fFbaF4bC0619080dC5c693CD"
        type: staticAggregationIsm
        modules:
          - address: "0x25FaF5270fF68572d5eCe14Fe9e29ac38E9266c1"
            type: merkleRootMultisigIsm
            validators:
              - "0x25B3A88f7CfD3C9F7d7e32b295673A16a6Ddbd91"
              - "0x4bce180DAC6da60D0f3a2Bdf036fFe9004F944C1"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x424478f999844bA8b6981Ac9Cad6Afd0dcB3cD29"
            type: messageIdMultisigIsm
            validators:
              - "0x25B3A88f7CfD3C9F7d7e32b295673A16a6Ddbd91"
              - "0x4bce180DAC6da60D0f3a2Bdf036fFe9004F944C1"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      linea:
        address: "0xe1aD314a8b9c18Eb3998606fc2b4B93e177c352a"
        type: staticAggregationIsm
        modules:
          - address: "0x56fe143F50885102968f8fb959aEBa822A55B5aD"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xf2d5409a59E0F5aE7635aFf73685624904A77d94"
            threshold: 2
          - address: "0xa278DA11F3afAC5Edcc0B03f641A172c9982EbA9"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xf2d5409a59E0F5aE7635aFf73685624904A77d94"
            threshold: 2
        threshold: 1
      mint:
        address: "0x5D342bC04BCd823e93bD36023927880a315c0B86"
        type: staticAggregationIsm
        modules:
          - address: "0x70b471964C81f5BEC7f8Fa8765F5f5631f1dBFe4"
            type: merkleRootMultisigIsm
            validators:
              - "0x0230505530B80186f8CdccFaF9993eb97AebE98a"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xFed01cCdd7a65E8A6ad867b7Fb03b9eB47777Ac9"
            threshold: 2
          - address: "0xe361aaa6d5530DC220A9dF300787f32A309A8FC9"
            type: messageIdMultisigIsm
            validators:
              - "0x0230505530B80186f8CdccFaF9993eb97AebE98a"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xFed01cCdd7a65E8A6ad867b7Fb03b9eB47777Ac9"
            threshold: 2
        threshold: 1
      mantapacific:
        address: "0x00cfc19eb97AbDcF3A724C771850b54a8c4616b4"
        type: staticAggregationIsm
        modules:
          - address: "0x424A84Dd6810Dd4430F4266e7F0F65160370D41e"
            type: merkleRootMultisigIsm
            validators:
              - "0x14025fE092f5f8a401DD9819704D9072196D2125"
              - "0x25b9A0961C51e74fd83295293Bc029131bF1E05A"
              - "0x42B6De2edBaA62C2eA2309Ad85D20B3E37d38Acf"
              - "0x521a3e6Bf8D24809fde1C1FD3494A859a16f132c"
              - "0x8e668c97Ad76D0e28375275c41ece4972AB8a5bc"
              - "0xa0eE95e280D46C14921e524B075d0C341e7ad1C8"
              - "0xcc9a0b6dE7Fe314bd99223687d784730a75bB957"
            threshold: 5
          - address: "0x59926F2e2C0fBd7a76b5CD101CfFF0b2A0A4e67F"
            type: messageIdMultisigIsm
            validators:
              - "0x14025fE092f5f8a401DD9819704D9072196D2125"
              - "0x25b9A0961C51e74fd83295293Bc029131bF1E05A"
              - "0x42B6De2edBaA62C2eA2309Ad85D20B3E37d38Acf"
              - "0x521a3e6Bf8D24809fde1C1FD3494A859a16f132c"
              - "0x8e668c97Ad76D0e28375275c41ece4972AB8a5bc"
              - "0xa0eE95e280D46C14921e524B075d0C341e7ad1C8"
              - "0xcc9a0b6dE7Fe314bd99223687d784730a75bB957"
            threshold: 5
        threshold: 1
      lukso:
        address: "0x47e9DB6Be0908c118aF6C90Ce4d9D8d03271616d"
        type: staticAggregationIsm
        modules:
          - address: "0x53B6B9976Ff98D09129D2bE14485AA79f5A7d7E2"
            type: messageIdMultisigIsm
            validators:
              - "0x101cE77261245140A0871f9407d6233C8230Ec47"
              - "0xa5E953701dCddC5b958b5DeFB677A829d908df6d"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xa5206D8649D8246FFd665a741d8A4e67497c5c08"
            type: merkleRootMultisigIsm
            validators:
              - "0x101cE77261245140A0871f9407d6233C8230Ec47"
              - "0xa5E953701dCddC5b958b5DeFB677A829d908df6d"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      inevm:
        address: "0x64DF9a02380bcd9d88b771771319818bff2d9664"
        type: staticAggregationIsm
        modules:
          - address: "0x2Fd7A5a277875C5d25237C4de796276B5852c0B8"
            type: merkleRootMultisigIsm
            validators:
              - "0x6B1d09A97b813D53e9D4b7523DA36604C0B52242"
              - "0x9aB11F38a609940153850Df611C9a2175dCfFe0f"
              - "0xF9e35Ee88e4448a3673b4676A4E153e3584a08EB"
            threshold: 2
          - address: "0x503b801999C335fAC2C306675DA8451799C52809"
            type: messageIdMultisigIsm
            validators:
              - "0x6B1d09A97b813D53e9D4b7523DA36604C0B52242"
              - "0x9aB11F38a609940153850Df611C9a2175dCfFe0f"
              - "0xF9e35Ee88e4448a3673b4676A4E153e3584a08EB"
            threshold: 2
        threshold: 1
      lisk:
        address: "0xdc595d3B0f16865183f901973A1f8ecE740C1091"
        type: staticAggregationIsm
        modules:
          - address: "0x2b6dD595594bEbF769aDD5f2819440B38ce1D8Ee"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xc0B282Aa5baC43Fee83CF71dC3dD1797C1090Ea5"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x4C9CAd3f24579508EA668c783fE51c97267E9dc1"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xc0B282Aa5baC43Fee83CF71dC3dD1797C1090Ea5"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      gnosis:
        address: "0xe81E0DE948916d812E9Df47df8B1c2D16DcFBf11"
        type: staticAggregationIsm
        modules:
          - address: "0x1Bf7Da2c41f3f21734117148ffA256cf26D4B4D9"
            type: merkleRootMultisigIsm
            validators:
              - "0x19fB7E04A1Be6b39b6966a0b0C60B929A93eD672"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0xd4dF66a859585678f2Ea8357161D896BE19cC1Ca"
            threshold: 3
          - address: "0xf8664453F6EaC09E1F867434369BD5Cd33a5AB65"
            type: messageIdMultisigIsm
            validators:
              - "0x19fB7E04A1Be6b39b6966a0b0C60B929A93eD672"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0xd4dF66a859585678f2Ea8357161D896BE19cC1Ca"
            threshold: 3
        threshold: 1
      moonbeam:
        address: "0x8627C149C148a114F85a0A5bA9Fb6Be62426129f"
        type: staticAggregationIsm
        modules:
          - address: "0x09a7055B314b3E1c70CB68f1DF737973574Ed5a8"
            type: messageIdMultisigIsm
            validators:
              - "0x2225e2F4e9221049456Da93B71D2De41f3B6b2A8"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x645428D198d2E76cbD9c1647F5c80740BB750B97"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
            threshold: 3
          - address: "0xe1839e0B3f11dAcF01a673C6b564609E33Ce288d"
            type: merkleRootMultisigIsm
            validators:
              - "0x2225e2F4e9221049456Da93B71D2De41f3B6b2A8"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x645428D198d2E76cbD9c1647F5c80740BB750B97"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
            threshold: 3
        threshold: 1
      merlin:
        address: "0x468381A96109eFC4C638f7C635347F5A5Db8cF93"
        type: staticAggregationIsm
        modules:
          - address: "0x535AE49AB4f9aB638CAceC714Dd4F4096A827d2A"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xc1D6600cB9326ED2198cC8C4Ba8d6668e8671247"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xAf85eF06FfD739fb6853e37B31ab2ce117B0F330"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xc1D6600cB9326ED2198cC8C4Ba8d6668e8671247"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      oortmainnet:
        address: "0x10B61b51e6BFD355296E8682227aCFCdd9498704"
        type: staticAggregationIsm
        modules:
          - address: "0xa9A99f51c3c44Ba6cF3313459498be0894C2216A"
            type: merkleRootMultisigIsm
            validators:
              - "0x032dE4f94676bF9314331e7D83E8Db4aC74c9E21"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x9B7ff56Cd9Aa69006F73f1C5b8C63390C706A5d7"
            threshold: 2
          - address: "0xb2164bFc8A82AaDf785fF1ABb39A48AE7Aaa983c"
            type: messageIdMultisigIsm
            validators:
              - "0x032dE4f94676bF9314331e7D83E8Db4aC74c9E21"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x9B7ff56Cd9Aa69006F73f1C5b8C63390C706A5d7"
            threshold: 2
        threshold: 1
      injective:
        address: "0x006a677cD9E24c9DCC906CA67fFf4d1dC1eCe984"
        type: staticAggregationIsm
        modules:
          - address: "0x3Deb42367eEf8b2D895689aa313f0eA9Eb679198"
            type: messageIdMultisigIsm
            validators:
              - "0x6B1d09A97b813D53e9D4b7523DA36604C0B52242"
              - "0x9e551B6694Bbd295D7d6e6A2540C7d41Ce70a3b9"
              - "0xbFB8911b72CfB138C7Ce517c57d9C691535DC517"
            threshold: 2
          - address: "0xA07B4d80F7d6ce8358A043dfaB23615969399174"
            type: merkleRootMultisigIsm
            validators:
              - "0x6B1d09A97b813D53e9D4b7523DA36604C0B52242"
              - "0x9e551B6694Bbd295D7d6e6A2540C7d41Ce70a3b9"
              - "0xbFB8911b72CfB138C7Ce517c57d9C691535DC517"
            threshold: 2
        threshold: 1
      xai:
        address: "0xdb3d2BCDFBddDC6e8B4924e163B062d0a1fde9C3"
        type: staticAggregationIsm
        modules:
          - address: "0x61e6Ae091F0705B366b2d8A41D23a0aCd37401d8"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xE993f01FEA86EB64cdA45ae5AF1d5BE40aC0c7E9"
            threshold: 2
          - address: "0xf3476a948aC6dC65603cD5ba202f222957fA292F"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xE993f01FEA86EB64cdA45ae5AF1d5BE40aC0c7E9"
            threshold: 2
        threshold: 1
      sanko:
        address: "0xB664a24Ff48FeBB374cF401206F9b8d57a492117"
        type: staticAggregationIsm
        modules:
          - address: "0x0C3d3cf5614d89A2dE473106057513BE60DC4107"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x795c37d5baBbc44094b084B0c89Ed9db9b5faE39"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0x7B4B984c973aAFE1E90FAA71291724e3214d4EB8"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x795c37d5baBbc44094b084B0c89Ed9db9b5faE39"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      real:
        address: "0xbafe5062F3e26b68525210611ED2E06B57Cf3A83"
        type: staticAggregationIsm
        modules:
          - address: "0x9101061A8F6BaCAC73D446c6916C29c278d34b97"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xAeBADd4998c70B05ce8715CF0C3cB8862fE0bEEC"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xb9Bc72551178482215c9b9FE220fC41F65D7aFED"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xAeBADd4998c70B05ce8715CF0C3cB8862fE0bEEC"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      optimism:
        address: "0x7840EBB87c5743Cb016411A02E6cF90879191d5c"
        type: staticAggregationIsm
        modules:
          - address: "0x4c4D775a96B23f30E7cAe0BD1D5612D46552a7Bb"
            type: messageIdMultisigIsm
            validators:
              - "0x20349eAdC6c72e94Ce38268b96692b1a5C20De4F"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x5B7D47b76C69740462432f6a5a0ca5005E014157"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
            threshold: 3
          - address: "0xD10Bd2E6879EB7D2f1a8755dC8bfEEdCfF6901B1"
            type: merkleRootMultisigIsm
            validators:
              - "0x20349eAdC6c72e94Ce38268b96692b1a5C20De4F"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
              - "0x5B7D47b76C69740462432f6a5a0ca5005E014157"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
            threshold: 3
        threshold: 1
      sei:
        address: "0x9F49fed50A28ba65C6F981227a523857efB2c157"
        type: staticAggregationIsm
        modules:
          - address: "0x5Df2696765CdFA41D063c0d69611c9933E2257f4"
            type: merkleRootMultisigIsm
            validators:
              - "0x101cE77261245140A0871f9407d6233C8230Ec47"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x9920D2dbF6C85ffc228fDc2e810bF895732c6aA5"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 3
          - address: "0x83938805b7153d3900f7769707f30275bdd3c507"
            type: messageIdMultisigIsm
            validators:
              - "0x101cE77261245140A0871f9407d6233C8230Ec47"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x9920D2dbF6C85ffc228fDc2e810bF895732c6aA5"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 3
        threshold: 1
      polygon:
        address: "0xCbC085eA3f391AEdb95B7029106479db7E874F72"
        type: staticAggregationIsm
        modules:
          - address: "0x0e7f8B0e67a0d791096Bc7F709A2bF662e276493"
            type: merkleRootMultisigIsm
            validators:
              - "0x008F24Cbb1cc30ad0f19f2516CA75730e37EfB5f"
              - "0x12ECb319C7f4e8ac5eB5226662aeB8528c5cefac"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
            threshold: 3
          - address: "0xF12042d3363D91781027bA85c9f2360c1BEC1Ce5"
            type: messageIdMultisigIsm
            validators:
              - "0x008F24Cbb1cc30ad0f19f2516CA75730e37EfB5f"
              - "0x12ECb319C7f4e8ac5eB5226662aeB8528c5cefac"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x5450447aeE7B544c462C9352bEF7cAD049B0C2Dc"
            threshold: 3
        threshold: 1
      proofofplay:
        address: "0x198f56eAcF85B4448Fc4ea8E3eF858F2cc1D3E67"
        type: staticAggregationIsm
        modules:
          - address: "0xcb73e2A35a01651c5dE84889c8DAF97Faf5DcfD3"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCda40BaA71970a06E5f55e306474dE5CA4E21C3b"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xeCE9C2C73b6A5ce8051259795C293072ce757E9c"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCda40BaA71970a06E5f55e306474dE5CA4E21C3b"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      osmosis:
        address: "0x3d66e818390E38535867962d68f0f03A50E13985"
        type: staticAggregationIsm
        modules:
          - address: "0x1422C74e198dF82B86aC74E0e97C193C68003B52"
            type: merkleRootMultisigIsm
            validators:
              - "0xeA483aF11c19fa41B16c31D1534c2A486a92BCAc"
            threshold: 1
          - address: "0xCe6010d22dA470835cB83D304541F69411473433"
            type: messageIdMultisigIsm
            validators:
              - "0xeA483aF11c19fa41B16c31D1534c2A486a92BCAc"
            threshold: 1
        threshold: 1
      redstone:
        address: "0xDA1F627ba6b587085B98690AE36C605118150237"
        type: staticAggregationIsm
        modules:
          - address: "0x99A0967E109FA9de08D7bb17bE46c07AA0599C06"
            type: messageIdMultisigIsm
            validators:
              - "0x1400b9737007f7978D8B4BBAFB4a69C83f0641A7"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xadD005598860BE5199D31Fa7607f401302C286ce"
            type: merkleRootMultisigIsm
            validators:
              - "0x1400b9737007f7978D8B4BBAFB4a69C83f0641A7"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      xlayer:
        address: "0xB2F04E7e4a3571c8Ae97C7EB5F229Ca5819A69f6"
        type: staticAggregationIsm
        modules:
          - address: "0x23a6392c019a9C9e6bE9640F6975D64bd1049d3F"
            type: messageIdMultisigIsm
            validators:
              - "0xa2Ae7C594703e988F23d97220717c513Db638ea3"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xfed056cC0967F5BC9C6350F6C42eE97d3983394d"
            threshold: 2
          - address: "0xB72cC1de8f9c0d2303219F8B0BB459265129caDC"
            type: merkleRootMultisigIsm
            validators:
              - "0xa2Ae7C594703e988F23d97220717c513Db638ea3"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xfed056cC0967F5BC9C6350F6C42eE97d3983394d"
            threshold: 2
        threshold: 1
      solanamainnet:
        address: "0xCe5C9067029CF9153fD8a12051f9B33FE85E15bb"
        type: staticAggregationIsm
        modules:
          - address: "0xD82870437b61579caEFA1E00A7Cdc84a08111393"
            type: messageIdMultisigIsm
            validators:
              - "0x28464752829b3ea59a497FCa0BDfF575c534C3ff"
              - "0x2b7514a2f77bD86bBF093fe6bB67d8611F51c659"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0xCb6bCbD0DE155072a7fF486d9d7286B0F71DcC2D"
              - "0xd90EA26fF731D967C5Ea660851F7d63Cb04ab820"
            threshold: 3
          - address: "0xa22F6827f358c51f7a00540942EFf30B61A92bB9"
            type: merkleRootMultisigIsm
            validators:
              - "0x28464752829b3ea59a497FCa0BDfF575c534C3ff"
              - "0x2b7514a2f77bD86bBF093fe6bB67d8611F51c659"
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0xCb6bCbD0DE155072a7fF486d9d7286B0F71DcC2D"
              - "0xd90EA26fF731D967C5Ea660851F7d63Cb04ab820"
            threshold: 3
        threshold: 1
      scroll:
        address: "0x989671F68D9A700FcDA0cceee10275a8cd0E1DE8"
        type: staticAggregationIsm
        modules:
          - address: "0x11eB2723A4147E3C4740A5D385567d3d3fe6F598"
            type: merkleRootMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0xaD557170a9f2f21C35E03de07CB30dcbcc3DfF63"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xBAc4ac39F1D8b5eF15f26fDB1294a7c9aBa3f948"
            threshold: 3
          - address: "0x866a137F5A237d7507FD41B9566fA084d2163264"
            type: messageIdMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0xaD557170a9f2f21C35E03de07CB30dcbcc3DfF63"
              - "0xb3AC35d3988bca8C2fFD195b1c6bee18536B317b"
              - "0xBAc4ac39F1D8b5eF15f26fDB1294a7c9aBa3f948"
            threshold: 3
        threshold: 1
      zoramainnet:
        address: "0x895fDAd9388cb2F3e279db8d3e98BbcC9a75690a"
        type: staticAggregationIsm
        modules:
          - address: "0xE67adaF94a7fA1a7A688e432e2b2cFCBba869Dbe"
            type: merkleRootMultisigIsm
            validators:
              - "0x35130945b625BB69B28AEe902A3b9a76FA67125F"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x7089B6352d37d23FB05A7fee4229C78E038fBa09"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 3
          - address: "0xf160Ac90f1933216ED9B425ae2bC2A61466Eb54c"
            type: messageIdMultisigIsm
            validators:
              - "0x35130945b625BB69B28AEe902A3b9a76FA67125F"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0x7089B6352d37d23FB05A7fee4229C78E038fBa09"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 3
        threshold: 1
      tangle:
        address: "0x577769094BCDe5228E6eb3999EC74543168e44A3"
        type: staticAggregationIsm
        modules:
          - address: "0x19B252ccEC97048e193265f2a18555f22a044976"
            type: merkleRootMultisigIsm
            validators:
              - "0x1EE52CBbFACd7dcB0ba4E91EfAa6FBc61602b15B"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xe271ef9A6E312540f099a378865432fA73f26689"
            threshold: 2
          - address: "0x639d3FC910dAa86149f072E4dF6501AcbBC9944A"
            type: messageIdMultisigIsm
            validators:
              - "0x1EE52CBbFACd7dcB0ba4E91EfAa6FBc61602b15B"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xe271ef9A6E312540f099a378865432fA73f26689"
            threshold: 2
        threshold: 1
      shibarium:
        address: "0x3E4943999744A8EC2086C7c0cD3e7b2F2D2C925b"
        type: staticAggregationIsm
        modules:
          - address: "0x24566f5EE6206ad0f0b5b0AB4720671535F2414C"
            type: messageIdMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xfA33391EE38597CbeEF72CCdE8c9e13E01e78521"
            threshold: 2
          - address: "0x607de20B65E9624C78fB80611215a1cc5D47ff5B"
            type: merkleRootMultisigIsm
            validators:
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
              - "0xfA33391EE38597CbeEF72CCdE8c9e13E01e78521"
            threshold: 2
        threshold: 1
      worldchain:
        address: "0xf0b40b26D638979793a53F4A7699A43e021599ea"
        type: staticAggregationIsm
        modules:
          - address: "0xF3F6A1D9aC6F93AC615B01700b8C3324f8925266"
            type: messageIdMultisigIsm
            validators:
              - "0x11E2a683e83617f186614071E422B857256a9aae"
              - "0x31048785845325B22817448B68d08F8a8Fe36854"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
          - address: "0xc8d029A2CdCb6853a8502f201D1331FefFAA152C"
            type: merkleRootMultisigIsm
            validators:
              - "0x11E2a683e83617f186614071E422B857256a9aae"
              - "0x31048785845325B22817448B68d08F8a8Fe36854"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 2
        threshold: 1
      viction:
        address: "0xb10200e048FafADb50AF2286C01cb1466F4fccC0"
        type: staticAggregationIsm
        modules:
          - address: "0x42DF3BD0dEaC280900463F2016D962fE33E16A07"
            type: messageIdMultisigIsm
            validators:
              - "0x1F87C368f8e05A85eF9126d984a980A20930cB9C"
              - "0x4E53dA92cD5Bf0a032b6B4614b986926456756A7"
              - "0xa3F93FE365bF99F431D8fdE740B140615e24f99B"
            threshold: 2
          - address: "0xc056cC12752A206C02ffD02bF4ab903b8DF95ca6"
            type: merkleRootMultisigIsm
            validators:
              - "0x1F87C368f8e05A85eF9126d984a980A20930cB9C"
              - "0x4E53dA92cD5Bf0a032b6B4614b986926456756A7"
              - "0xa3F93FE365bF99F431D8fdE740B140615e24f99B"
            threshold: 2
        threshold: 1
      taiko:
        address: "0x566aDaaBDB44B17669CEE92246355345F68Ca2F5"
        type: staticAggregationIsm
        modules:
          - address: "0x48Ec78b5AA7a746B02f62718308d374f18643fFA"
            type: merkleRootMultisigIsm
            validators:
              - "0x2F007c82672F2Bb97227D4e3F80Ac481bfB40A2a"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xa930073C8f2D0B2F7423ea32293E0d1362E65D79"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 3
          - address: "0x6ec58342BbCDBC500B956fc7241BbFA67e798C14"
            type: messageIdMultisigIsm
            validators:
              - "0x2F007c82672F2Bb97227D4e3F80Ac481bfB40A2a"
              - "0x4f977A59fDc2D9e39f6D780A84D5b4aDD1495a36"
              - "0xa930073C8f2D0B2F7423ea32293E0d1362E65D79"
              - "0xCF0211faFBb91FD9D06D7E306B30032DC3A1934f"
            threshold: 3
        threshold: 1
      polygonzkevm:
        address: "0x3B0e1e627b1326281ad7F18Fc61d286A80e1361E"
        type: staticAggregationIsm
        modules:
          - address: "0xC20a62901dCA12aF1C23314D8dF457B016369aC6"
            type: merkleRootMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x865818Fe1dB986036d5FD0466dCd462562436d1A"
              - "0x86f2A44592bB98dA766E880cfd70D3bbb295e61a"
            threshold: 2
          - address: "0xf77f393d957457A7d120927778587B1521Fcd444"
            type: messageIdMultisigIsm
            validators:
              - "0x38C7A4ca1273ead2E867d096aDBCDD0e2AcB21D8"
              - "0x865818Fe1dB986036d5FD0466dCd462562436d1A"
              - "0x86f2A44592bB98dA766E880cfd70D3bbb295e61a"
            threshold: 2
        threshold: 1
      zircuit:
        address: "0xb3A1CC4423366C51cf175df4a13992eAd0D41865"
        type: staticAggregationIsm
        modules:
          - address: "0x797B7C2379a140910d2EF919Fc21F1845cbB6633"
            type: messageIdMultisigIsm
            validators:
              - "0x0180444c9342BD672867Df1432eb3dA354413a6E"
              - "0x169ec400CC758fEf3df6A0D6C51FBC6CdD1015bb"
              - "0x1da9176C2CE5cC7115340496fa7D1800a98911CE"
              - "0x7aC6584c068eb2A72d4Db82A7B7cd5AB34044061"
            threshold: 3
          - address: "0xb028D6dC92F18829De6C3a73A7beBdE2Fd7B6E57"
            type: merkleRootMultisigIsm
            validators:
              - "0x0180444c9342BD672867Df1432eb3dA354413a6E"
              - "0x169ec400CC758fEf3df6A0D6C51FBC6CdD1015bb"
              - "0x1da9176C2CE5cC7115340496fa7D1800a98911CE"
              - "0x7aC6584c068eb2A72d4Db82A7B7cd5AB34044061"
            threshold: 3
        threshold: 1
threshold: 2
```

Reply:
smart